inspirehep/utils/normalizers.py

```diff
@@ -22,19 +22,22 @@
 
 from __future__ import absolute_import, division, print_function
 
+import logging
```

Comment:
This change happened due to [this comment](https://github.com/inspirehep/inspire-next/pull/2299#pullrequestreview-35949298). 

Which of these two methods for logging, do you guys think I should use?